CODEOWNERS

```diff
@@ -1,88 +1,250 @@
+# Instructions for CODEOWNERS file format and automatic build failure notifications:
+# https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
+
+#########
+# Specification
+#########
+
+# PRLabel: %Cognitive Services
 /dev/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+
 /specification/adp/ @Azure/adp
+
+# PRLabel: %Analysis Services
 /specification/analysisservices/ @taiwu
+
+# PRLabel: %Alerts Management
 /specification/alertsmanagement/ @ofirmanor @olalavi @erangon @orieldar @ilaizi @shakednai1 @khaboasb @orenhor
+
+# PRLabel: %API Management
 /specification/apimanagement/ @promoisha @solankisamir
+
+# PRLabel: %Insights
 /specification/applicationinsights/ @alexeldeib
+
+# PRLabel: %Monitor - Exporter
 /specification/applicationinsights/data-plane/Monitor.Exporters/ @ramthi @trask @hectorhdzg @lzchen @Azure/api-stewardship-board
+
+
 /specification/asazure/ @athipp
+
+# PRLabel: %Authorization
 /specification/authorization/ @darshanhs90 @stankovski
+
+# PRLabel: %Automation
 /specification/automation/ @vrdmr
+
 /specification/azsadmin/ @deathly809
+
+# PRLabel: %Batch
 /specification/batch/ @paterasMSFT @dpwatrous
+
+# PRLabel: %BatchAI
 /specification/batchai/ @alexanderyukhanov
+
+# PRLabel: %Billing
 /specification/billing/ @wilcobmsft @asarkar84
+
+# PRLabel: %Network - CDN
 /specification/cdn/ @jorinmejia @yunhemsft @jessicl-ms @sinadell @rrahulms @t-bzhan
+
+# PRLabel: %Cognitive Services
 /specification/cognitiveservices/ @felixwa @yangyuan
+
+# PRLabel: %Cognitive - Language
 /specification/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+
+# PRLabel: %Compute
 /specification/compute/ @bilaakpan-ms @sandido @dkulkarni-ms @haagha @madewithsmiles @MS-syh2qs @grizzlytheodore @hyonholee @mabhard @danielli90 @smotwani @ppatwa @vikramd-ms @savyasachisamal @yunusm @ZhidongPeng @nkuchta @maheshnemichand @najams @changov
+
 /specification/consumption/ @kjeur @panda-wang
+
+# PRLabel: %Container Instances
 /specification/containerinstance/ @novinc
+
+# PRLabel: %Container Registry
 /specification/containerregistry/ @djyou
+
+# PRLabel: %Container Service
 /specification/containerservice/ @palma21 @weinong @seguler @alvinli222 @justindavies @matthchr @robbiezhang @paulgmiller @yizhang4321 @circy9 @qike-ms
+
+# PRLabel: %Cosmos
 /specification/cosmos-db/ @dmakwana @MehaKaushik @wmengmsft @shurd
+
+# PRLabel: %Customer Insights
 /specification/customer-insights/ @tjlvtao
+
+# PRLabel: %Data Factory
 /specification/datafactory/ @Frey-Wang @zhangyd2015 @davidzhaoyue
+
+# PRLabel: %Data Lake Analytics
 /specification/datalake-analytics/ @ro-joowan
+
+# PRLabel: %Data Lake Store
 /specification/datalake-store/ @ro-joowan
+
+# PRLabel: %Data Migration
 /specification/datamigration/ @hitenjava
+
 /specification/deploymentmanager/ @netrock
+
```

Comment:
```suggestion

# PRLabel: %Device Update
/specification/deviceupdate/data-plane/ @mikekistler
```

Reply:
For data-plane, you can use "@Azure/api-stewardship-board" so reviewer is shared to the team (if you do not intend to review for all deviceupdate changes).